### DISCUSSION TITLE
Stable?

### DISCUSSION THREAD

**Discussion by snowy-jaguar** [psycopg/psycopg#228] [General]
I'm assuming that since [psycopg.org/](https://www.psycopg.org/) is saying that it's *under development* there is no stable version of psycopg3?

**Comment by dvarrazzo** [comment]
No, it's stable. The website needs some update.

**Reply by snowy-jaguar** [reply]
Oh okay, guess I'll update the install I just did from V2 to V3 (if that's even how your versioning is going)
